### PR TITLE
feat(tools): enable german docs download

### PR DESCRIPTION
**Pull request by naomi-lgbt**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This will enable the downloading of the German translations for our contributing documentation. The documentation is at about 63% approved, but I received a request in chat to enable this.

AFTER MERGE:

I will manually run the docs download to get the german files, then PR to add it to the nav bar.

### CODE REVIEW

File: .github/workflows/crowdin-download.docs.yml [hunk 1]
```diff
@@ -235,6 +235,41 @@ jobs:
           # Uncomment below to debug
           # dryrun_action: true
 
+        env:
+          GITHUB_TOKEN: ${{ secrets.CROWDIN_CAMPERBOT_PAT }}
+          CROWDIN_PROJECT_ID: ${{ secrets.CROWDIN_PROJECT_ID_DOCS }}
+          CROWDIN_PERSONAL_TOKEN:
+            ${{ secrets.CROWDIN_CAMPERBOT_SERVICE_TOKEN }}
```
Comment (ShaunSHamilton) [inline .github/workflows/crowdin-download.docs.yml:242]: Silly Question

Do we really need to constantly redeclare the `env` variables for each step?

Comment (nhcarrigan) [reply .github/workflows/crowdin-download.docs.yml:242]: y'know...
No, we don't. We could declare them top level instead.

**Comment by stephanduesterhoeft** [comment]
The documentation is now 100% approved

**Review by ojeytonwilliams — APPROVED** [review]
@nhcarrigan did you want to pull the env stuff to the top before we forget?  Either way, this seems good.

**Review by moT01 — APPROVED** [review]
LGTM 🎉 Do we want to go with this as-is or add @ShaunSHamilton's suggestion here? I would vote for a separate PR.

edit: didn't see that Oliver asked the same question

**Comment by nhcarrigan** [comment]
Yeah we can do that in a separate PR. It'll be a tad bit noisy.

**Pull request merged**